src/SlackClientFactory.ts

```diff
@@ -49,7 +49,7 @@ export class SlackClientFactory {
                     // non-ideal way to detect calls to slack.
                     webLog.debug.bind(webLog);
                     if (!this.onRemoteCall) { return; }
-                    const match = /apiCall\('([\w\.]+)'\) start/.exec(msg);
```

Comment:
Again, pretty sure this is meant to be a literal. We should double slash.

Reply:
The dot character is always literal inside a `[]` list.

Reply:
oh, didn't know that.